spark3/src/main/java/org/apache/iceberg/spark/source/SparkMicroBatchStream.java

```diff
@@ -106,7 +107,9 @@
     this.splitOpenFileCost = Spark3Util.propertyAsLong(
         options, SparkReadOptions.FILE_OPEN_COST, tableSplitOpenFileCost);
 
-    InitialOffsetStore initialOffsetStore = new InitialOffsetStore(table, checkpointLocation);
+    this.fromTimestamp = Spark3Util.propertyAsLong(options, SparkReadOptions.STREAM_FROM_TIMESTAMP, -1L);
```

Comment:
We typically use `null` to signal that a value is not present instead of -1.

Reply:
In this case, I think `long fromTimestamp` will present as  `Long fromTimestamp`?

Reply:
We expect the behavior that if the fromTimestamp is not set, we wanna start from the timestamp 0. If we wanna set the fromTimestamp to null, we will reset it to -1 or 0 later. I think it is fine to use -1 as default value here

Reply:
Yes, if you want to set a field to null, you can't use the primitive type `long`. You would need to use the boxed type `Long`.